tests/test_pi_io.py

```diff
@@ -39,5 +39,3 @@ def test_setup_only_called_once_per_pin(self):
         self.assertEqual(mock_GPIO.setup.call_args_list,
                          [((pin_one, mock_GPIO.OUT),),
                           ((pin_two, mock_GPIO.OUT),)])
```

Comment:
Oh we should add a test case to make sure `close` works. I think really simple like

```python
     def test_close(self):
         mock_GPIO = mock.Mock()
         io = pi_io.IO(mock_GPIO)
         io.close()
         mock_GPIO.cleanup.assert_called_once()
```